app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/migration/NameIdOrgController.scala

```diff
@@ -132,13 +132,13 @@ class NameIDOrgController @Inject() (
   private def populateOkView(
     nameUtrViewModel: Option[NameIdOrganisationMatchModel],
     organisationType: String,
-    conf: Configuration,
```

Comment:
We should probably just tweak the name of this parameter to match the type (viewModel? vm? take your pick(